src/main/java/com/mate/bookstore/dto/BookDto.java

```diff
@@ -1,21 +1,28 @@
 package com.mate.bookstore.dto;
 
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class BookDto {
     private Long id;
 
+    @NotBlank(message = "Title is required")
     private String title;
 
+    @NotBlank(message = "Author is required")
     private String author;
 
+    @NotBlank(message = "isbn is required")
```

Comment:
The message for the `@NotBlank` annotation on the 'isbn' field should be capitalized for consistency with other messages.
```suggestion
    @NotBlank(message = "ISBN is required")
```